<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QWidget" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="insulinUnit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PowerBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="InsulinSpinBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="TimeDisplay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateDisplay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="insulinLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Insulin On Board</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BatterySpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="InsulinOnBoard">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="glucoseUnit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mmol/L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OptionsBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>OPTIONS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bolusbtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>BOLUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="BatteryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Battery</string>
   </property>
  </widget>
  <widget class="QListWidget" name="GraphWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="GlucoseLevel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChargeBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
